ui-modules/app-inspector/app/views/main/inspect/summary/summary.template.html

```diff
@@ -86,13 +86,28 @@ <h2 ng-if="vm.entity" class="entity-name">
 
         <div ng-if="vm.location.items">
             <div ng-if="vm.location.items.length !== 0 || !vm.location.multi">
-                <p>This entity is running on <em><ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': '{} location', 'other': '{} locations'}"></ng-pluralize></em></p>
-                <ul>
-                    <li ng-repeat="item in vm.location.items track by item.id">
-                        <strong>{{item.config['spec.final'] | specToLabel:vm.metadata}}</strong>
-                        <span ng-if="item.config.availabilityZone">in <code>{{item.config.availabilityZone}}</code> availability zone</span>
-                    </li>
-                </ul>
+                <p>This entity is associated with <ng-pluralize count="vm.location.items.length" when="{'0': 'no locations', 'one': 'this location ', 'other': 'these locations'}"></ng-pluralize></p>
+                    <div ng-repeat="item in vm.location.items track by item.id">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <h3 class="panel-title">Location ID: <strong>{{item.id | specToLabel:vm.metadata}}</strong></h3>
+                            </div>
+                            <div class="panel-body">
+                                <ul>
+                                    <li ng-if="item.name"> <span>Name:  {{item.name | specToLabel:vm.metadata}}</span> </li>
+                                    <li ng-if="item.config['spec.final']"> <span> Environment:  {{item.config['spec.final'] | specToLabel:vm.metadata}}</span> </li>
+                                    <li ng-if="item.spec"> <span>Spec:  {{item.spec | specToLabel:vm.metadata}}</span> </li>
+                                    <li ng-if="item.type"> <span>Type:  {{item.type | specToLabel:vm.metadata}}</span> </li>
+                                    <li ng-if="item.config['address']"> <span>Address:  {{item.config['address'] | specToLabel:vm.metadata}}</span> </li>
+                                    <li ng-if="item.config['endpoint']"> <span>Endpoint:  {{item.config['endpoint'] | specToLabel:vm.metadata}}</span> </li>
```

Comment:
The following code is unreachable

```
... | specToLabel:vm.metadata}}
```

because of `ng-if`

Reply:
To simplify? e.g.
```
<li ng-if="item.name"> <span>Name:  {{item.name}}</span> </li>
```

Reply:
@algairim  I have no idea how but ... `specToLabelFilter` function in  `summary.controller.js` is called. :|